source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -14,18 +14,43 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## Reason #1. [Community](https://emberjs.com/community) 🏆
+
+One great reason to use Ember is the [community](https://emberjs.com/community) that you are/will be a part of. Ember provides a wide platform for everyone to be an active learner and contributor. Where will you discover yourself next?
+
+This year, we surveyed 10 different ways to [learn](https://emberjs.com/ember-community-survey-2019/#MS_Q103) and [contribute](https://emberjs.com/ember-community-survey-2019/#MS_Q112)!
+
+Our 5 most favorite places to learn: (drumrolls, please)
+
+- [Documentations](https://guides.emberjs.com/release/) (71%)
+- [Ember.js blog](https://blog.emberjs.com/) (54%)
+- [RFCs](https://github.com/emberjs/rfcs) (52%)
+- Code samples (51%)
```

Comment:
Thoughts on linking to twiddle?

Reply:
I'm okay with both including the link and leaving as is.

I didn't include a link because code samples, to me, may be a part of a blog post or documentation. I wasn't sure what link would make a good representative, and hadn't thought about Ember Twiddle.